### PR TITLE
add solidus_easypost file

### LINKED ISSUE #30: rake railties:install:migrations
Also running rake railties:install:migrations doesn't generate any migrations.

I'm on the following versions:

ruby 2.5.3
Rails 5.2.2
Solidus 2.7.1

### PR DESCRIPTION
**Pull request by peterberkenbosch**
the extension is expected to have a `solidus_easypost` file to load the
engine and other libraries. This commit introduces that file that is
a simple wrapper around the existing `spree_easypost` file.

This fixes the problem with not loading migrations. ( fixes #30 )

### CODE REVIEW

**Review by aitbw — CHANGES_REQUESTED** [review]
Thanks a lot for this PR, @peterberkenbosch ! :raised_hands: 

But I have a few questions:

* Wouldn't it be better if we renamed [this file](https://github.com/solidusio-contrib/solidus_easypost/blob/master/lib/spree_easypost.rb) accordingly?

* If that's the case, shouldn't we rename the folder `spree_easypost` under `lib/` too?

* It seems that this extension is missing a `version.rb` file under `lib/spree_easypost/`, what do you think about adding said file on this PR as well?

**Comment by peterberkenbosch** [comment]
yes, I agree @aitbw, this was the quick fix mode but the best is to rename it. I'll add the version file too.

**Comment by gildardoperez** [comment]
@peterberkenbosch OMG. Awesome! Thank you!

**Comment by peterberkenbosch** [comment]
@aitbw renamed all and introduced version file. Decided to go to a `2.0` version since we renamed it and this would be a breaking change.

**Review by aitbw — APPROVED** [review]
Fantastic work, @peterberkenbosch :heart: :clap: thanks a lot for this patch! :bowing_woman:

**Comment by aitbw** [comment]
What do you think on this, @kennyadsl @jacobherrington ?

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The extension also adds a callback to the "ship" event on the `Shipment` model,
 
 ```ruby
   # config/initializers/easy_post.rb
-  Spree::EasyPost::configs[:purchase_labels?] = false
+  Solidus::EasyPost::configs[:purchase_labels?] = false
```
Comment (aitbw) [inline README.md]: This should be `CONFIGS` instead of `configs`, right?

Comment (peterberkenbosch) [reply README.md]: yes indeed! Good catch. update in a sec.

File: solidus_easypost.gemspec [hunk 2]
```diff
@@ -1,8 +1,12 @@
 # encoding: UTF-8
+lib = File.expand_path('../lib/', __FILE__)
```
Comment (kennyadsl) [inline solidus_easypost.gemspec]: I think you can just use `require_relative` here, [as it's done in Solidus](https://github.com/solidusio/solidus/blob/4a1e245f2f8d300de45bf4efc2b6a425b58f2b5c/solidus.gemspec#L3).

**Review by kennyadsl — COMMENTED** [review]
Thanks @peterberkenbosch, just left a suggestion 👍

**Review by jacobherrington — APPROVED** [review]
Thanks @peterberkenbosch!

File: solidus_easypost.gemspec (hunk 2 above)
Comment (peterberkenbosch) [reply solidus_easypost.gemspec]: much cleaner indeed. fixed :)

**Comment by aitbw** [comment]
Let's wait to reach (and fix) a consensus on solidusio/solidus#3074 before merging this.

**Comment by aitbw** [comment]
Re-running specs now that [solidusio/solidus#3078](https://github.com/solidusio/solidus/pull/3078) and [solidusio/solidus#3079](https://github.com/solidusio/solidus/pull/3079) are merged.

**Comment by peterberkenbosch** [comment]
I will rebase this on master when #33 is merged. Thanks @aitbw

**Comment by aitbw** [comment]
Done with #33, @peterberkenbosch :ok_hand:

**Comment by aitbw** [comment]
Re-running specs due to a Travis failure.

**Pull request merged**